app/controllers/EventsController.scala

```diff
@@ -75,4 +84,22 @@ trait EventsController extends BaseController {
       case _: UnknownVenueException => BadRequest(s"$venueParam is not a valid venue")
     }
   }
+
+  def allocations(eventId: String): Action[AnyContent] = Action.async { implicit request =>
+    assessorAllocationService.getAllocations(eventId).map { allocations =>
+      if (allocations.allocations.isEmpty) {
+        NotFound
```

Comment:
I think if there are no allocations for a particular event, it is not really an exceptional situation, it is perfectly possible. In that sense, I think returning empty list is normal. What do the others think?